imap/ctl_mboxlist.c

```diff
@@ -1007,6 +1007,9 @@ int main(int argc, char *argv[])
 
     case M_POPULATE:
         syslog(LOG_NOTICE, "%spopulating mupdate", warn_only ? "test " : "");
+#if defined __GNUC__ &&  __GNUC__ > 6
```

Comment:
Yeah, I don't really think one-line macros like this make debugging any harder (especially when it's a no-op line like this anyway), and encapsulating the nastiness makes the switch cases much more readable